### PR TITLE
Release 473 : Weekly Realease

### PR DESCRIPTION
**Pull request by mhbensghaier**

### CODE REVIEW

**Comment by mhbensghaier** [comment]
Requested changes for entreprise skin done

File: exo-platform/RELEASE-473.json [hunk 1]
```diff
@@ -0,0 +1,860 @@
...
+       "version":"16-M01",
+       "current_snapshot_version":"16-SNAPSHOT",
+       "next_snapshot_version":"16-SNAPSHOT",
+       "patches":"",
+       "patches_after_release":"",
+       "nexus_host":"exoplatform",
+       "nexus_staging_profile":"public"
+    }
+ }
+    {
```
Comment (vsellier) [inline exo-platform/RELEASE-473.json]: Please check the indentation of this block

File: exo-platform/RELEASE-473.json [hunk 2]
```diff
@@ -0,0 +1,860 @@
...
+       },
+       "git_organization":"exoplatform",
+       "labels":"addons",
+       "maven_property_version":"addon.exo.enterprise-skin.version",
+       "container": {
+        "image": "exoplatform/release:jdk8-maven33",
+        "step": 2
+      },
+       "release":{
+          "branch":"feature/enterprise-skin",
```
Comment (vsellier) [inline exo-platform/RELEASE-473.json]: The branch is now ``develop``

File: exo-platform/RELEASE-473.json [hunk 3]
```diff
@@ -0,0 +1,860 @@
...
+          "version":"5.0.0-M28",
```
Comment (vsellier) [inline exo-platform/RELEASE-473.json]: we are releasing the version ``5.0.0-M29``

File: exo-platform/RELEASE-473.json [hunk 4]
```diff
@@ -0,0 +1,860 @@
...
+          "current_snapshot_version":"1.0.x-enterprise-skin-SNAPSHOT",
+          "next_snapshot_version":"1.0.x-enterprise-skin-SNAPSHOT",
+          "patches":"",
```
Comment (vsellier) [inline exo-platform/RELEASE-473.json]: The current and next snapshots must be ``5.0.0-SNAPSHOT`` as declared on the pom.xml of the project : https://github.com/exoplatform/enterprise-skin/blob/develop/pom.xml#L32

Comment (mhbensghaier) [reply exo-platform/RELEASE-473.json]: changes done as requested

**Pull request merged**